src/dynarec/la64/dynarec_la64_f30f.c

```diff
@@ -331,6 +331,48 @@ uintptr_t dynarec64_F30F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int
                 SMWRITE2();
             }
             break;
+        case 0xB8:
+            INST_NAME("POPCNT Gd, Ed");
+            SETFLAGS(X_ALL, SF_SET);
+            SET_DFNONE();
+            nextop = F8;
+            v1 = fpu_get_scratch(dyn);
+            GETED(0);
+            GETGD;
+            if (MODREG) {
+                if (rex.w) {
+                    VXOR_V(v1, v1, v1);
+                    VINSGR2VR_D(v1, ed, 0);
+                } else {
+                    ZEROUP2(x4, ed); // need to clear uper part
+                    ed = x4;
+                    VXOR_V(v1, v1, v1);
+                    VINSGR2VR_D(v1, ed, 0);
+                }
+            } else {
+                if (rex.w) {
+                    SMREAD();
+                    addr = geted(dyn, addr, ninst, nextop, &ed, x1, x2, &fixedaddress, rex, NULL, 0, 0);
+                    FLD_D(v1, ed, fixedaddress);
+                } else {
+                    VXOR_V(v1, v1, v1);
+                    VINSGR2VR_D(v1, ed, 0);
+                }
+            }
+            CLEAR_FLAGS(x2);
+            BNEZ_MARK(ed);
```

Comment:
What this line is for? I suppose it’s for the flags calculation, but ed here is a memory address sometimes.